libs/superagent/app/models/response.py

```diff
@@ -67,6 +67,10 @@ class AgentInvoke(BaseModel):
     success: bool
     data: Any
 
+class AgentDatasourceRecency(BaseModel):
```

Comment:
Would suggest shoving it in the same `agents_rag` file alongside `AgentRAGInvoke`